tests/test_faster_rcnn.py

```diff
@@ -104,8 +99,8 @@ def test_faster_rcnn_train_all(mock_create, mock_loss, config, dataset):
         kfp_writer=kfp_writer,
     )
     checkpointer.save = MagicMock()
-    train_dataset = create_dataset(config, TRAIN)
-    val_dataset = create_dataset(config, VAL)
+    train_dataset = dataset
+    val_dataset = dataset
```

Comment:
Shall we test using both train and val dataset?

Reply:
I'm not quite sure what you mean. Functionally the test has not changed. The mocking would return the same dataset twice. Can you elaborate what you mean?

Reply:
In previous version, we set train_dataset and val_dataset into different datasets. Can you set `train_dataset = dataset_train` and `val_dataset = dataset_val`?

Reply:
I dont think this is the case due to the mocking of the @patch("datasetinsights.estimators.faster_rcnn.Dataset.create") This static method is called in create_dataset(config,TRAIN) and create_dataset(config,VAL). The mock makes sure that the Dataset.create static method only returns dataset fixture given in the argument dataset.

Reply:
Make sense.